tools/cldr-apps/js/src/index.js

```diff
@@ -2,13 +2,42 @@ import "./style.css";
 // This is the entrypoint for the new SurveyTool app
 
 import { createApp } from "vue";
-import About from "./views/About.vue";
 
-function showAbout(el) {
-  return createApp(About).mount(el);
+import AboutPanel from "./views/AboutPanel.vue";
+import WaitingPanel from "./views/WaitingPanel.vue";
+
+function show(component, el) {
+  const app = createApp(component);
```

Comment:
minor quibble: I would move show() down below the function that calls it

old-fashioned languages like Pascal required low-level functions at the top and high-level functions at the bottom, and that's also seen in some old-fashioned ways of writing JavaScript ("var foo = function..." vs modern "function foo") -- for me, at least, code is way easier to read with high-level at the top and subroutines at the bottom